common/match/pool.ts

```diff
@@ -534,17 +603,27 @@ export async function getPupilsToRequestInterest(pool: MatchPool): Promise<Pupil
 }
 
 export async function sendConfirmationRequests(pool: MatchPool) {
-    const pupils = await getPupilsToRequestInterest(pool);
+    const pupils = await getPupilsToContactNext(pool, 'confirmation-unknown');
     for (const pupil of pupils) {
         await requestInterestConfirmation(pupil);
     }
 }
 
+export async function addPupilScreenings(pool: MatchPool) {
```

Comment:
Optional parameter here too, expose via GraphQL to ADMIN (with parameter matchPool, screenings to send)